samples/dashboard/demos/climate/demo.js

```diff
@@ -4,66 +4,63 @@ const color1 = Highcharts.Color.parse('#39F');
 const color2 = Highcharts.Color.parse('#6C0');
 const color3 = Highcharts.Color.parse('#F00');
 const dataPool = new Dashboard.DataOnDemand();
-
+const dataScopes = {
+    'FD' : 'Days with fog',
+    'ID' : 'Days with ice',
+    'RR1' : 'Days with rain',
+    'TN' : 'Average temperature',
+    'TX' : 'Maximal temperature'
+};
+
+let citiesData;
 let cityGrid;
 let citySeries;
 let dataScope = 'TX';
-let worldTime = new Date(2010, 12, 25); 
+let worldCities;
+let worldDate = new Date(Date.UTC(2010, 11, 25)); 
 
 async function buildDashboard() {
-    const topology = await Promise
-        .resolve('https://code.highcharts.com/mapdata/custom/world.topo.json')
-        .then(fetch)
-        .then(response => response.json());
 
-    const citiesTable = await dataPool.getStoreTable('cities');
-    const cityClimate = await dataPool.getStore('Tokyo');
+    citiesData = await buildCitiesData();
 
-    let citiesMap;
-    let worldTable = await dataPool.getStoreTable(1262649600000);
+    const defaultCity = await dataPool.getStore('Tokyo');
 
     const dashboard = new Dashboard.Dashboard('container', {
         components: [{
             cell: 'time-range-selector',
             type: 'Highcharts',
             chartOptions: {
                 chart: {
-                    height: '100px',
+                    height: '60px',
+                },
+                credits: {
+                    enabled: false
                 },
                 series: [{
                     type: 'scatter',
                     name: 'Timeline',
                     data: buildDates(),
                     events: {
-                        click: function (e) {
-                            dataPool
-                                .getStoreTable(e.point.x)
-                                .then(table => {
-                                    worldTable = table;
-                                    citiesMap.setData(buildCitiesMap(
-                                        worldTable,
-                                        citiesTable
-                                    ));
-                                });
+                        click: async function (e) {
+                            worldDate = new Date(e.point.x);
+                            worldCities.setData(await buildCitiesMap());
                         }
                     },
                     tooltip: {
-                        footerFormat: void 0,
-                        headerFormat: void 0,
+                        footerFormat: '',
+                        headerFormat: '',
```

Comment:
Why we use empty string instead of default `undefined` ?

Reply:
Because this throws internal exceptions. Bug in maps.